dev-python/spyder-notebook/spyder-notebook-0.2.1.ebuild

```diff
@@ -0,0 +1,38 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{6,7} )
+
+inherit distutils-r1 virtualx
+
+DESCRIPTION="Jupyter notebook integration with Spyder"
+HOMEPAGE="https://github.com/spyder-ide/spyder-notebook"
+SRC_URI="https://github.com/spyder-ide/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+RDEPEND="dev-python/nbformat[${PYTHON_USEDEP}]
+	dev-python/notebook[${PYTHON_USEDEP}]
+	dev-python/psutil[${PYTHON_USEDEP}]
+	dev-python/QtPy[webkit,${PYTHON_USEDEP}]
```

Comment:
new package blocking deprecated Qt5WebKit cleanup really makes me sad.

Reply:
Not trying to step on to your toes, but why is it deprecated? Upstream looks pretty active to me, 
https://github.com/qtwebkit/qtwebkit/branches/active

Reply:
cf36947cb83d55e687b318bb04fdff1c59ab737c would this work for spyder-notebook as well? If not, it looks like an optional dep and it could be left out from this update...

Reply:
> Not trying to step on to your toes, but why is it deprecated? Upstream looks pretty active to me,

QtWebKit is deprecated since February 2015. What we are looking at right now is a fork that has not gotten further than alpha status, with one year of inactivity in between. That's not to say we aren't glad someone is doing the valuable work keeping it on life support while upstreams are porting away.

Reply:
> new package blocking deprecated Qt5WebKit cleanup really makes me sad.

spyder-notebook, seems to work just fine without QtPy[webkit] and PyQt5[webkit]. Tests pass (though still not inside emerge, it dumps core), and a quick test inside spyder confirms the plugin loads correctly.

I think the reason I added QtPy[webkit] initially was because the file `spyder-notebook-0.2.1/spyder_notebook/utils/templates//weclome.html` contains lines with `-webkit-......`. But as I said it works fine without webkit support built into QtPy, so I'm not sure what those lines are about.

Reply:
that's a false positive then indeed, there's no QtWebKit involved in html files.